cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.v2v, pytest.mark.manual]
+
+
+@pytest.mark.tier(1)
+def test_osp_vmware_67_test_vm_name_with_punycode_characters():
```

Comment:
We are not supporting this anymore now, developers decided to add pre-flight checks for this [BZ1685948](https://bugzilla.redhat.com/show_bug.cgi?id=1685948), there is one internal email thread too.